ext/pcntl/pcntl.c

```diff
@@ -1400,19 +1405,18 @@ PHP_FUNCTION(pcntl_async_signals)
 PHP_FUNCTION(pcntl_unshare)
 {
 	zend_long flags;
-	int ret;
 
 	ZEND_PARSE_PARAMETERS_START(1, 1)
 		Z_PARAM_LONG(flags)
 	ZEND_PARSE_PARAMETERS_END();
 
-	ret = unshare(flags);
-	if (ret == -1) {
+	if (unshare(flags) == -1) {
 		PCNTL_G(last_error) = errno;
 		switch (errno) {
 #ifdef EINVAL
 			case EINVAL:
-				php_error_docref(NULL, E_WARNING, "Error %d: Invalid flag specified", errno);
+				zend_argument_value_error(1, "must be a valid identifier flag");
```

Comment:
```suggestion
				zend_argument_value_error(1, "must be a combination of CLONE_* flags");
```
I don't the the "identifier flag" terminology makes sense here.